README.md

```diff
@@ -79,14 +79,14 @@ Running spring app locally and db in docker
 
 1. `cd backend`
 1. Run `docker-compose up -d db`
-1. Run `gradle bootRun --args='--spring.profiles.active=dev'`
+1. Run `./gradle bootRun --args='--spring.profiles.active=dev'`
```

Comment:
These should actually be `./gradlew`. Technically `gradle` will work if you have gradle installed on your system, but `./gradlew` is a wrapper script (that's what the w stands for) that downloads a specific gradle version to your system if it doesn't already have it, so it is the more compatible command

Reply:
Update in 8bf68e6